common/changes/@microsoft/rush/raihle-patch-4256_2023-08-03-15-04.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Improve version resolution logic in common/scripts/install-run.js (#4256)",
+      "type": "patch"
```

Comment:
```suggestion
      "type": "none"
```

The Rush projects are lockstep versioned, so the next bump kind isn't controlled by the changefiles. See https://rushjs.io/pages/maintainer/publishing/#3-version-policy

Reply:
Oh! Here I thought I had absent-mindedly selected the wrong change-type and went to fix it...